### PR TITLE
[#5] Introduction of Unit Testing to the project 

### PR DESCRIPTION
**Pull request by haydende**
This only includes a single GET test against http://hostname:3000/, and will serve as a base for additional tests

### CODE REVIEW

**Review by AdamJ335-T2 — APPROVED** [review]
Looks good

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM python:3.11-slim
 WORKDIR /code
-COPY src/app/requirements.txt /code
+COPY requirements.txt /code
 RUN pip3 install -r requirements.txt --no-cache-dir
 COPY . /code
 EXPOSE 3000
-ENTRYPOINT ["gunicorn", "-b", "0.0.0.0:3000", "-k", "gevent", "-w", "1", "--chdir", "src/app", "app:app"]
+ENTRYPOINT ["gunicorn", "-b", "0.0.0.0:3000", "-k", "gevent", "-w", "1", "src.app.app:app"]
```
Comment (jrsmth-tier2) [inline Dockerfile:7]: Lovely

File: src/app/app.py [hunk 2]
```diff
@@ -10,38 +10,38 @@
...
-# Initialise messages
-messages = Messages(from_root('resources', 'messages.properties'))
-
-
-# Register blueprint
-app.register_blueprint(archbishop.construct_blueprint(slack_events_adapter, config, messages, redis))
-
-
-# Let's go!
+# Only get to here when running directly
```
Comment (jrsmth-tier2) [inline src/app/app.py:45]: So when will this get used?

**Review by jrsmth-tier2 — APPROVED** [review]

File: src/app/app.py (hunk 2 above)
Comment (haydende) [reply src/app/app.py:45]: I think only if you can get it started on your host

**Pull request merged**